feature/s3/transfermanager/types.go

```diff
@@ -0,0 +1,205 @@
+package transfermanager
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/s3/types"
+)
+
+// ChecksumAlgorithm indicates the algorithm used to create the checksum for the object
+type ChecksumAlgorithm string
+
+// Enum values for ChecksumAlgorithm
+const (
+	ChecksumAlgorithmCrc32  ChecksumAlgorithm = "CRC32"
+	ChecksumAlgorithmCrc32c ChecksumAlgorithm = "CRC32C"
```

Comment:
You can omit the `ChecksumAlgorithm` type declaration on every line but the first, compiler figures it out